webapp/src/projects.tsx

```diff
@@ -713,11 +724,78 @@ export class ProjectsDetail extends data.Component<ProjectsDetailProps, Projects
             ((/^https:\/\//i.test(url)) || (/^\//i.test(url)) ? url : '');
     }
 
+    protected getClickLabel(cardType: string) {
+        const { youTubeId } = this.props;
+        let clickLabel = lf("Show Instructions");
+        if (cardType == "tutorial")
+            clickLabel = lf("Start Tutorial");
+        else if (cardType == "codeExample" || cardType == "example")
+            clickLabel = lf("Open Example");
+        else if (cardType == "forumUrl")
+            clickLabel = lf("Open in Forum");
+        else if (cardType == "template")
+            clickLabel = lf("New Project");
+        else if (youTubeId)
+            clickLabel = lf("Play Video");
+        return clickLabel;
+    }
+
+    protected getActionIcon(onClick: any, type: string, action?: pxt.CodeCardAction): JSX.Element {
+        const { youTubeId } = this.props;
+        let icon = "file text";
+        switch (type) {
+            case "tutorial":
+            case "example":
+                icon = "xicon blocks"
+                if (action) icon = action.editor == "py" ? "xicon python" : "xicon js";
+                break;
+            case "codeExample":
+                icon = (action && action.editor == "py") ? "xicon python" : "xicon js";
+                break;
+            case "forumUrl":
+                icon = "comments"
+                break;
+            case "template":
+            default:
+                if (youTubeId) icon = "video camera";
+                break;
+        }
+        return this.isLink() && type != "example" // TODO (shakao)  migrate forumurl to otherAction json in md
```

Comment:
since the icon and text are both links you have to tab twice per option; probably `tabindex-1` for the icon so it just goes to the button?